packages/EasyUtils/tests/AbstractMathTestCase.php

```diff
@@ -43,28 +43,89 @@ public function provideAbsData(): array
     }
 
     /**
-     * @return mixed[]
+     * @return iterable<mixed>
      *
-     * @see testCompSucceeds
+     * @see testCompareSucceeds
      */
-    public function provideCompData(): array
+    public function provideCompareData(): iterable
     {
-        return [
-            [
-                'leftOperand' => '10000000',
-                'rightOperand' => '10000001',
-                'result' => -1,
-            ],
-            [
-                'leftOperand' => '10000000',
-                'rightOperand' => '10000000',
-                'result' => 0,
-            ],
-            [
-                'leftOperand' => '10000001',
-                'rightOperand' => '10000000',
-                'result' => 1,
-            ],
+        yield [
+            'leftOperand' => '10000000',
+            'rightOperand' => '10000001',
+            'compareMethod' => 'greaterOrEqualTo',
+            'result' => false,
+        ];
+        yield [
+            'leftOperand' => '10000000',
+            'rightOperand' => '10000000',
+            'compareMethod' => 'greaterOrEqualTo',
+            'result' => true,
+        ];
+        yield [
+            'leftOperand' => '10000001',
+            'rightOperand' => '10000000',
+            'compareMethod' => 'greaterOrEqualTo',
+            'result' => true,
+        ];
+        yield [
+            'leftOperand' => '10000000',
+            'rightOperand' => '10000000',
+            'compareMethod' => 'equalTo',
+            'result' => true,
+        ];
+        yield [
```

Comment:
I thought you gonna test each method using 3 pairs of operands, leftOperant > rightOperand, leftOperand = rightOperand, leftOperand < rightOperand. But this one is lacking leftOperand > rightOperand. Next one is lacking leftOperand = rightOperand. Also, check lessThan.

Reply:
done